docs/tutorials/dev_add_test_group.md

```diff
@@ -0,0 +1,2466 @@
+
+Our new `YetAnotherChannel` class defines the test group, but so far it
+doesn't have any test cases in it.  We'll come back and add them later in the
+tutorial.  Before we add a test case, let's make `polaris` aware that the
+test group exists. To do that, we need to open 
+[polaris/ocean/__init__.py](https://github.com/E3SM-Project/polaris/blob/main/polaris/ocean/__init__.py),
+add an import for the new test group, and add an instance of the test group to the list of test
+groups in the ocean core:
+
+```{code-block} python
+:emphasize-lines: 2, 18
+
+from polaris import Component
+from polaris.ocean.tests.baroclinic_channel import BaroclinicChannel
+from polaris.ocean.tests.global_convergence import GlobalConvergence
+from polaris.ocean.tests.yet_another_channel import YetAnotherChannel
+
+
+class Ocean(Component):
+    """
+    The collection of all test case for the MPAS-Ocean core
+    """
+
+    def __init__(self):
+        """
+        Construct the collection of MPAS-Ocean test cases
+        """
+        super().__init__(name='ocean')
+
+        # please keep these in alphabetical order
+        self.add_test_group(BaroclinicChannel(component=self))
+        self.add_test_group(GlobalConvergence(component=self))
+        self.add_test_group(YetAnotherChannel(component=self))
+```
+
+We make an instance of the `YetAnotherChannel` class and we immediately add
+it to the `Ocean` core's list of test groups.  That's all we need to do.  Now
+`polaris` knows about the test group.
+
+(dev-tutorial-add-test-group-add-default)=
+
+## Adding a "default" test case
+
+We'll add a test case called `default` to `yet_another_channel` by making a
+`default` package within `polaris/ocean/tests/yet_another_channel`.  First,
+we make the directory `polaris/ocean/tests/yet_another_channel/default`, then
+we add an empty `__init__.py` file into it. As a starting point, we'll create
+a new `Default` class in this file that descends from the
+{py:class}`polaris.TestCase` base class (take a look at
+`polaris/testcase.py` if you want to see the contents of `TestCase` if
+you're interested).
+
+```python
+from polaris import TestCase
+
+
+class Default(TestCase):
+    """
+    The default test case for the "yet another channel" test group simply creates
+    the mesh and initial condition, then performs a short forward run on 4
+    cores.
+    """
+
+    def __init__(self, test_group):
+        """
+        Create the test case
+
+        Parameters
+        ----------
+        test_group : polaris.ocean.tests.yet_another_channel.YetAnotherChannel
+            The test group that this test case belongs to
+        """
+        name = 'default'
+        super().__init__(test_group=test_group, name=name)
+```
+
+As a starting point, we just pass along the test group (`YetAnotherChannel`)
+this test case belongs to on to the base class's constructor
+(`super().__init__()`) and give the test case a name, `default`.
+
```

Comment:
```suggestion
Even though this test case doesn't do anything, we can list the tests and make
sure your new one shows up:

```bash
$ polaris list

Testcases:
...
   9: ocean/yet_another_channel/default
[end ```bash]

If they don't show up, you probably missed a step (adding the test group to the
component or the test case to the test group).  If you get import errors or
syntax errors, you'll need to fix those first.

```

Reply:
I don't think we've added `Default` to the test group at this point, so this isn't going to do anything yet.  Your comment lower down might be where that's appropriate.

Reply:
Or I can edit the tutorial to add the test case to the test group here, and list it even though it doesn't do anything.  That's a little more back-and-forth between test group, test case and steps but I think that's okay.

Reply:
Yes, I think it's best to add it to the default case first here, without the resolution argument. I personally think it's easier to understand what's going on if you have a chance to build the test group/case up slowly with a lot of back-and-forth.

Reply:
I've done this and will push changes soon.